src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const PRICE_PER_DAY = 40;
+
+  let totalPrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    totalPrice -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    totalPrice -= SHORT_TERM_DISCOUNT;
+  }
```

Comment:
Prefer `if` with `return` over `if else` to simplify the conditions and make the code more readable. Also, don't add `else` after `if` with `return` as the code after it won't be executed anyway (CHECKLIST items 4 and 5).